test/integration/default_elements.cc

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2017 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <iostream>
+#include <string>
+#include <gtest/gtest.h>
+
+#include "sdf/SDFImpl.hh"
+#include "sdf/parser.hh"
+#include "sdf/Frame.hh"
+#include "sdf/Model.hh"
+#include "sdf/Root.hh"
+#include "sdf/World.hh"
+#include "sdf/Filesystem.hh"
+#include "test_config.h"
+
+//////////////////////////////////////////////////
+TEST(ExplicitlySetInFile, EmptyRoadSphCoords)
+{
+  const std::string test_file =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "sdf",
+        "empty_road_sph_coords.sdf");
+
+  sdf::Root root;
+  sdf::Errors errors = root.Load(test_file);
+  EXPECT_TRUE(errors.empty());
+
+  sdf::ElementPtr elementPtr = root.Element();
+  EXPECT_TRUE(root.Element()->GetExplicitlySetInFile());
+
+  elementPtr = elementPtr->GetFirstElement();
```

Comment:
Can we use the [`GetElement`](https://github.com/osrf/sdformat/blob/sdf9/include/sdf/Element.hh#L328L337) function, to have more visibility on which element we are trying to grab?

Reply:
Doesn't `GetElement` require the element to have a name? some of the elements we are testing don't have one. Also all pointer variables have now specific names for clarity on the element it's being obtained.

Reply:
Got it, thanks!